README.md

```diff
@@ -1142,7 +1165,7 @@ Sample result:
 
 The result object exposes the following properties:
 
-- `last_modified`: the [collection's timestamp](http://kinto.readthedocs.io/en/stable/api/1.x/timestamps.html). This value is opaque and should be reused as is, eg. passing it as a `since` option (see [Generic bucket and collection options](#generic-bucket-and-collection-options))
+- `last_modified`: the [collection's timestamp](http://kinto.readthedocs.io/en/stable/api/1.x/timestamps.html).
```

Comment:
Maybe we should note that it is not the same as the timestamps you get from `getCollectionsTimestamp()` etc.

Reply:
Actually, it is the same :) I'll add a note